### PR TITLE
BLD: fix "python setup.py develop".  Closes gh-196.

### PR DESCRIPTION
**Pull request by rgommers**
Also fixes ``pip install -e``, which calls ``develop``.
Does not fix ``python setup.py build_ext``, that should still be
preceded by ``python setup.py build_clib``.

This won't win a beauty contest, but seems to be the only way to preserve the separation of clib from the build to help C code coverage.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -147,6 +149,53 @@ def write_version_py(filename='pywt/version.py'):
...
+    Note: if you want to build in-place with ``python setup.py build_ext``,
+    that will only work if you first do ``python setup.py build_clib``.
+
+    """
+    def install_for_development(self):
+        self.run_command('egg_info')
+
+        # Build extensions in-place (the next 7 lines are the monkeypatch)
+        import glob
+        hitlist = glob.glob(os.path.join('build', '*', 'libc_wt.a'))
```
Comment (grlee77) [inline setup.py]: do we need a check for Windows here too?
I think it may be c_wt.lib there.

**Comment by grlee77** [comment]
If I get a chance, I may try from a windows PC later (see the comment I made), but otherwise this looks good.

File: setup.py (hunk 1 above)
Comment (rgommers) [inline setup.py]: good point, changed to ``libc_wt.*``

**Comment by grlee77** [comment]
Thanks, @rgommers .

**Pull request merged**